src/pages/LinkList/components/LinksGrid/interfaces/Link.ts

```diff
@@ -0,0 +1,7 @@
+export interface Link {
+  id: string;
+  link: string;
+  status: string;
```

Comment:
This is a great place to use discriminated unions:

```suggestion
  status: "Await" | "Approved" | "Rejected";
```

By the way, we tend to use lower case for these kinds of values:

```suggestion
  status: "await" | "approved" | "rejected";
```

Reply:
It could be good, but those three values ​​are values ​​I composed for a possible server-side return.

Reply:
I would fix the server side then. In general, we are trying to use lowercase values in our JSONs.

If fixing it on the server side is not an option, you could also use discriminated unions to represent the values: `"Await" | "Approved" | "Rejected"`

Reply:
I would like to add this change when we are more certain about the starcture and data, or we can modify this now assuming that this will work in the future. But we must take into account that it is not enough to modify only this line, this would cause an error with the type in the data file. It would be necessary to import the link interface to that file and modify all the values ​​of the simulated status attributes. What do you prefer?

Reply:
No estoy seguro de que cambio hablás, si de usar lowercase o si hacer el tipo más específico.

Una opción es dejarlo así y exponer derecho lo que viene del backend, eso a la larga va a cambiar porque vas a necesitar mostrarlo en español o en inglés.

Por eso, tipificarlo y restringirlo es una forma de documentar que es lo que tu sistema espera. De esa forma, quien hace el backend sabe mejor que es lo que tiene que hacer. 

Básicamente, definir acá exactamente cuales van a ser los valores aceptados es lo mismo que usar `string` o `number` en lugar de `any`, al poner tipos, estás documentando y restringiendo lo que el sistema puede hacer y en general eso es bueno.

> It would be necessary to import the link interface to that file and modify all the values ​​of the simulated status attributes

Creo que el cambio es menor, me parece que no vas a necesitar agregar muchos imports. Tal vez un casteo en donde tomás los datos del JSON.

Reply:
hable de ambos cambios, siendo modificar el mock algo mas tedioso, pero si crees mejor los convierto a lowercase

Reply:
Mmm, yo lo veía como un cambio simple, no pensé que fuera tedioso, por ahí se me escapa algo, en ese caso confío en tu criterio.

Reply:
mi criterio era fuera de scope, pero lo sumo porque no quiero llevar la contra 😄

Reply:
done